centreon/doc/API/v23.10/Configuration/Dashboard/Schema/Share/DashboardShareContact.Add.yaml

```diff
@@ -0,0 +1,17 @@
+type: object
```

Comment:
Please avoid mixing CamelCase and snakeCase

Reply:
Sorry, but can you precise what you meant ? because it's `camelCase`, `PascalCase` or `snake_case`.
I don't see if you referenced the content or the file.

Reply:
My bad, I was referring to the file name which is a mix of PascalCase and dotCase

Reply:
Can be corrected later